apl-vault-wallet/src/main/java/com/apollocurrency/aplwallet/vault/service/KMSv1.java

```diff
@@ -0,0 +1,38 @@
+package com.apollocurrency.aplwallet.vault.service;
+
+import com.apollocurrency.aplwallet.api.dto.vault.ExportKeyStore;
+import com.apollocurrency.aplwallet.vault.model.EthWalletKey;
+import com.apollocurrency.aplwallet.vault.model.KMSResponseStatus;
+import com.apollocurrency.aplwallet.vault.model.WalletKeysInfo;
+
+import java.util.List;
+
+public interface KMSv1 {
```

Comment:
I guess it's not a suitable class name. 
I suppose it's not a KMS, it's something like an adapter to KMS service